src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const FEMALE = 'f';
+  const notMarried = 'I am not married';
+
+  const partner = sex === FEMALE ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        person.age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+        )
+      }
+
+      {
+        isMarried
+          ? (
+            <p className="Person__partner">
+              {`${partnerName} is my ${partner}`}
+            </p>
```

Comment:
Don't repeat yourself: the outer paragraph is the same for both cases
Change just content depending on `isMarried` variable